pkg/operator/v1helpers/core_getters.go

```diff
@@ -100,3 +116,26 @@ func (g combinedSecretInterface) List(opts metav1.ListOptions) (*corev1.SecretLi
 	}
 	return ret, nil
 }
+
+func validateGetOptions(options metav1.GetOptions) error {
+	return validateResourceVersion(options.ResourceVersion)
+}
+
+func listOptionsToSelector(options metav1.ListOptions) (labels.Selector, error) {
+	if len(options.FieldSelector) > 0 || options.Watch || len(options.Continue) > 0 {
```

Comment:
what about other fields in `ListOptions` struct? (https://github.com/openshift/library-go/blob/96bee2747f6cf4443917046d7341b01dc5d5e1b4/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/types.go#L334)

if we don't want to repeat the fields we could create an empty list, rewrite the supported fields and compare the diff.